pgcli/packages/sqlcompletion.py

```diff
@@ -106,13 +126,13 @@ def suggest_special(text):
 
     if cmd == text:
         # Trying to complete the special command itself
-        return [{'type': 'special'}]
+        return Special(),
```

Comment:
I'd still prefer to have parens around single value tuples. It's way too easy to miss the trailing comma.